src/Composer/ComposerWindow.vala

```diff
@@ -19,18 +19,27 @@
  */
 
 public class Mail.ComposerWindow : Gtk.ApplicationWindow {
+    public ComposeRequest template { get; construct; }
+
     public ComposerWindow (Gtk.Window parent) {
+        this.from_request (parent, null);
```

Comment:
elementary code style avoids `this` where possible. I realize that isn't in the code style guide. Sorry, our mistake!

Reply:
So, I don't think it's avoidable in this instance (calling one constructor from another)? I tried a couple other syntaxes, and they didn't work.